include/jemalloc/internal/rtree_types.h

```diff
@@ -25,11 +26,17 @@ typedef struct rtree_s rtree_t;
 #define RTREE_HEIGHT_MAX						\
     ((1U << (LG_SIZEOF_PTR+3)) / RTREE_BITS_PER_LEVEL)
 
+/*
+ * Number of leafkey/leaf pairs to cache.  Each entry supports an entire leaf,
+ * so the cache hit rate is typically high even with a small number of entries.
+ * Two entries is enough to keep the hit rate high even when an arena uses a
+ * combination of dss and mmap.
+ */
+#define RTREE_CTX_NCACHE 2
```

Comment:
We have 8, <= 12, and 16 (mine) as poll results, which definitely helps narrow the range. :smile:  I'll make it 8 for now and we can reevaluate closer to 5.0.0 release if we see rtree performance issues.

Do we need double-cacheline alignement (128) to assure the following cacheline is pulled in automatically, or does it happen regardless of whether the cacheline is at an even multiple offset?